sbin/Release.sh

```diff
@@ -13,84 +13,40 @@
 # limitations under the License.
 #
 
-TIMESTAMP="$(date +'%Y%m%d')"
 npm install
-for f in OpenJDK*.tar.gz
+# Expect a naming convention for build artifacts to follow this pattern:
+# OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
+# Examples: OpenJDK8_x64_Windows_201813060547.zip, 
+#           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
+#           OpenJDK10_aarch64_Linux_201813060547.sha256.txt
+#           OpenJDKamber_x64_Linux_201813061304.tar.gz	
+for f in OpenJDK*
 do
-  case $f in
-    *Linux*)
-      OS=Linux && EXT=tar.gz
-      case $f in
-        *x64*)
-        ARCH=x64 ;;
-        *s390x*)
-        ARCH=s390x ;;
-        *ppc64le*)
-        ARCH=ppc64le ;;
-        *aarch64*)
-        ARCH=aarch64 ;;
-    esac ;;
-    *Mac*)
-    OS=Mac && ARCH=x64 && EXT=tar.gz ;;
-    *AIX*)
-    OS=AIX && ARCH=ppc64 && EXT=tar.gz ;;
-  esac
-  if [ "$REPO" == "releases" ]; then
-    mv $f "Open${VERSION}_${ARCH}_${OS}_${TAG}.${EXT}"
-    elif [ "$REPO" == "nightly" ]; then
-    mv $f "Open${VERSION}_${ARCH}_${OS}_$TIMESTAMP.${EXT}"
+#              1)VERSION 2)ARCH         3)OS           4)TS_TAG       5)EXTENSION 
+  regex="OpenJDK([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+).(tar.gz|sha256.txt|zip)";
```

Comment:
is the convention to use the local keyword in front of a local bash var?

Reply:
I was not sure if we(Adopt) followed a programming convention for shell scripts, based on your question, I looked up a commonly followed one (https://google.github.io/styleguide/shell.xml), and note that they use local for variables in functions.  So will leave as is, but in general will try to comply with the google style guide for shell scripting.

Reply:
thanks for the pointer to shellcheck too